plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/internal/CubridMessages.properties

```diff
@@ -0,0 +1,3 @@
+dialog_struct_create_procedure_container=Owner;
+dialog_struct_create_procedure_label_name=Name;
+dialog_struct_create_procedure_combo_type=Type;

```

Comment:
No `;` is required - it's a key-value file with newline as a delimiter. These values will end up containing `;` in the UI.

Reply:
I have just fixed this.